composer.json

```diff
@@ -13,7 +13,8 @@
         "google/protobuf": "^3.12.2",
         "guzzlehttp/promises": "^1.3",
         "guzzlehttp/psr7": "^1.7.0||^2",
-        "google/common-protos": "^1.0||^2.0"
+        "google/common-protos": "^1.0||^2.0",
```

Comment:
While it's not absolutely required, it would be best to allow `^3.0` of `google/common-protos` here as well, as that version will not have any of the operations protos in it.

See https://github.com/googleapis/common-protos-php/pull/41